### PR TITLE
add cleopatra & geostatista packages

### PR DESCRIPTION
**Pull request by MAfarrag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MAfarrag** [comment]
@isuruf  @xhochy can you please review here all checks have passed, Thanks

File: recipes/cleopatra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
```
Comment (ndevenish) [inline recipes/cleopatra/meta.yaml:23]: Is pip actually a runtime dependency or does this come from the fact that the environment file on the repo uses it to install more?

File: recipes/geostatista/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
```
Comment (ndevenish) [inline recipes/geostatista/meta.yaml:23]: Same here?
```suggestion
```

File: recipes/cleopatra/meta.yaml (hunk 1 above)
Comment (MAfarrag) [reply recipes/cleopatra/meta.yaml:23]: yes because i use it in the environment file in the repo

File: recipes/geostatista/meta.yaml (hunk 2 above)
Comment (MAfarrag) [reply recipes/geostatista/meta.yaml:23]: same here

**Comment by MAfarrag** [comment]
@ndevenish loguru package is a conda-forge check here [https://anaconda.org/conda-forge/loguru](https://anaconda.org/conda-forge/loguru) and yes it is required.

**Comment by ndevenish** [comment]
Oh, my apologies, I must have searched the wrong name. If it's required then you probably want to add to the `run:` section here?

**Comment by MAfarrag** [comment]
@ndevenish I have added the package and all checks are passed.

**Comment by MAfarrag** [comment]
@conda-forge/help-python review here please all checks have passed. Thanks

**Comment by hmaarrfk** [comment]
1 day turnaround is really tight. I suggest you consider to start to maintain your channel too. with pure python packages, it is really easy to build it up even if you don't use full feedstocks . Once packages get accepted, you can delete them from your channel, or archive them.

**Comment by MAfarrag** [comment]
@hmaarrfk thanks a lot for your help.

**Pull request merged**